frontend/src/pages/AnnouncementsPage.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { ApplicationState } from '../store'
+import { Typography } from '@material-ui/core'
+import { selectAnnouncements } from '../models/announcements'
+import { AnnouncementCard } from '../components/AnnouncementCard'
+import { Container } from '../components/Container'
+import { Title } from '../components/Title'
+import { Icon } from '../components/Icon'
+
+export const AnnouncementsPage = () => {
+  const enabled = useSelector((state: ApplicationState) => selectAnnouncements(state))
+
+  return (
+    <Container
```

Comment:
are you to show this component if `enabled` doesn't have data?

Reply:
Yes we would show the page anyway. It just would be empty until the announcements are loaded. I rename enabled to annoucements. It used to be that I was checking enabled, but Benoit said he's checking that cloud side so I removed it.